bundle/manifests/falcon-operator.clusterserviceversion.yaml

```diff
@@ -0,0 +1,181 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  annotations:
+    alm-examples: |-
+      [
+        {
+          "apiVersion": "falcon.crowdstrike.com/v1alpha1",
+          "kind": "FalconConfig",
+          "metadata": {
+            "name": "falconconfig-sample"
+          },
+          "spec": {
+            "foo": "bar"
+          }
+        }
+      ]
+    capabilities: Basic Install
+    operators.operatorframework.io/builder: operator-sdk-v1.5.0
+    operators.operatorframework.io/project_layout: go.kubebuilder.io/v3
+  name: falcon-operator.v0.0.1
+  namespace: placeholder
```

Comment:
IIRC, @redhatrises mentioned that namespaces should not be hard coded. 

@redhatrises what do you recommend here?

Reply:
All this code is generated from `make bundle`. Namespaces shouldn't be hard coded for helm charts which is different than a golang operator.